client/ayon_houdini/api/lib.py

```diff
@@ -729,23 +725,25 @@ def get_current_context_template_data_with_folder_attrs():
 
     # get context specific vars
     folder_attributes = folder_entity["attrib"]
+    task_attributes = task_entity["attrib"]
 
     # compute `frameStartHandle` and `frameEndHandle`
-    frame_start = folder_attributes.get("frameStart")
-    frame_end = folder_attributes.get("frameEnd")
-    handle_start = folder_attributes.get("handleStart")
-    handle_end = folder_attributes.get("handleEnd")
-    if frame_start is not None and handle_start is not None:
-        folder_attributes["frameStartHandle"] = frame_start - handle_start
-
-    if frame_end is not None and handle_end is not None:
-        folder_attributes["frameEndHandle"] = frame_end + handle_end
+    for attributes in [folder_attributes, task_attributes]:
+        frame_start = attributes.get("frameStart")
+        frame_end = attributes.get("frameEnd")
+        handle_start = attributes.get("handleStart")
+        handle_end = attributes.get("handleEnd")
+        if frame_start is not None and handle_start is not None:
+            attributes["frameStartHandle"] = frame_start - handle_start
+        if frame_end is not None and handle_end is not None:
+            attributes["frameEndHandle"] = frame_end + handle_end
 
     template_data = get_template_data(
         project_entity, folder_entity, task_entity, host_name
     )
     template_data["root"] = anatomy.roots
     template_data["folderAttributes"] = folder_attributes
+    template_data["taskAttributes"] = task_attributes
```

Comment:
Where are `folderAttributes` and `taskAttributes` actually used?

Reply:
These I believe are used by the feature to update houdini session variables based on context changes. So you could e.g. store houdini-specific variables like the folder or tasks's frame range to be used dynamically inside parms.

Basically related to this feature by @MustafaJafar I believe: https://github.com/ynput/OpenPype/pull/5651

Reply:
> Where are `folderAttributes` and `taskAttributes` actually used?

It's used in this feature to manage Houdini variables. 
![image](https://github.com/ynput/ayon-houdini/assets/20871534/e53608fa-8001-4d78-968b-444cf994619f)

and this how it look like in Houdini.
![image](https://github.com/ynput/ayon-houdini/assets/20871534/25877766-1325-434c-a6b5-f08c35924818)